api/src/test/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExhaustionControllerTest.java

```diff
@@ -79,7 +78,7 @@ class CloudTaskInitialCreditsExhaustionControllerTest {
 
   @Autowired UserDao userDao;
   @Autowired WorkspaceDao workspaceDao;
-  @Autowired WorkspaceFreeTierUsageDao workspaceFreeTierUsageDao;
+  @Autowired WorkspaceFreeTierUsageDao workspaceInitialCreditsUsageDao;
```

Comment:
I chose not to change this class, because then its name would mismatch its DB table